dolphin/api/v1/storages.py

```diff
@@ -132,8 +132,21 @@ def create(self, req, body):
     def update(self, req, id, body):
         return dict(name="Storage 4")
 
+    @wsgi.response(202)
     def delete(self, req, id):
-        return webob.Response(status_int=http_client.ACCEPTED)
+        ctxt = req.environ['dolphin.context']
+        try:
+            device = db.storage_get(ctxt, id)
+        except Exception as e:
```

Comment:
may be we can add specific exception
ex:
            except exception.StorageNotFound as e:
            LOG.error(e)
            raise exc.HTTPNotFound(explanation=e.msg)

Reply:
Done